internal/app/storage/shortener_db.go

```diff
@@ -0,0 +1,46 @@
+package storage
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type DBStorage struct {
+	conn *pgxpool.Conn
+}
+
+func NewDBStorage(c *pgxpool.Conn) *DBStorage {
+	return &DBStorage{conn: c}
+}
+
+func (st *DBStorage) AddLink(full string, short string) {
+	_, _ = st.conn.Exec(context.Background(), "INSERT INTO links(long,short) VALUES ($1,$2);", full, short)
```

Comment:
What if we would like to use another database manager system?

Reply:
So we would better use squirrel :)